ajax/libs/botui/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "botui",
+  "description": "A JS library to build the UI for your bot",
+  "homepage": "https://botui.org",
+  "filename": "botui.min.js",
+  "author": {
+    "name": "Moin Uddin",
+    "url": "https://moin.im"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moinism/botui.git"
+  },
+  "keywords": [
+    "bot",
+    "chat",
+    "conversational",
+    "interface",
+    "js",
+    "ui"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
```

Comment:
Let's use **npm auto-update**
Because I found that there are 14 releases versions from npm, and 4 releases versions from GitHub.
As usual, we prefer to use more releases versions as the auto-update option.
Thank you.